.github/workflows/check-prs.yml

```
@@ -0,0 +1,80 @@
+jobs:
+  check-prs:
+    name: Close Inactive PRs and Notify Issues
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+      issues: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)